QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,132 @@
+This document is a declaration of software quality for the `rcpputils` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+# `rcpputils` Quality Declaration
+
+The package `rcpputils` claims to be in the **Quality Level 4** category.
+
+Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Requirements for Quality Level 4 in REP-2004](https://www.ros.org/reps/rep-2004.html).
```

Comment:
```suggestion
Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Quality Categories in REP-2004](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#package-quality-categories) of the ROS2 developer guide.
```

Reply:
Since the rep content is slated for removal from the developer's guide, I'm choosing to link to the future home of the rep. Hopefully it gets merged soon...